practiceQuestion2.rb

```diff
@@ -0,0 +1,18 @@
+no_of_element = gets().chomp().to_i
+arr = Array.new(no_of_element)
+no_of_element.times do |item|
+  num=gets.chomp().to_i;
+  arr[item]=num;
+end
+frequency = Hash.new
+for e in arr do
+   if !frequency.key?(e)
+      frequency[e]=1;
+   else
+     frequency[e]=1+frequency[e];
+   end
+end
```

Comment:
Easy way to do this:
```suggestion
frequency = Hash.new(0)
for e in arr do
   frequency[e] = 1 + frequency[e]
end
```